x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/overview_tab_content.tsx

```diff
@@ -0,0 +1,352 @@
+                      </EuiCodeBlock>
+                      <EuiSpacer size="s" />
+                    </EuiDescriptionListDescription>
+                  </EuiDescriptionList>
+                </EuiFlexItem>
+                {module.query !== undefined ? (
+                  <EuiFlexItem grow={false}>
+                    <EuiFlexGroup gutterSize="m" alignItems="center" justifyContent="flexEnd">
+                      <EuiFlexItem grow={false}>
+                        <EuiText size={'xs'}>
+                          <FormattedMessage
+                            id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.dataRecognizerHelpButtonLabel"
+                            defaultMessage="Check indices to see which can be used to run jobs in this module?"
+                          />
+                        </EuiText>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        <span>
+                          <EuiButton
+                            fill
+                            isDisabled={runningDataRecognizer}
+                            isLoading={runningDataRecognizer}
+                            color={'primary'}
+                            onClick={runDataRecongizer}
+                            size="s"
+                          >
+                            <FormattedMessage
+                              id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.runDataRecognizerButtonLabel"
+                              defaultMessage="Run data recognizer"
+                            />
+                          </EuiButton>
+                        </span>
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  </EuiFlexItem>
+                ) : null}
+                {matchingDataViews.length ? (
+                  <EuiFlexItem grow={false}>
+                    <DataViewsTable matchingDataViews={matchingDataViews} moduleId={module.id} />
+                  </EuiFlexItem>
+                ) : null}
+              </EuiFlexGroup>
+            ) : (
+              <EuiEmptyPrompt
+                css={{ margin: '10px' }}
+                color="primary"
+                iconType={(isLogoObject(module.logo) ? module.logo?.icon : module.logo) as string}
+                title={
+                  <h2>
+                    <FormattedMessage
+                      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.unableToUseModuleTitle"
+                      defaultMessage="You cannot create these jobs here"
+                    />
+                  </h2>
+                }
+                body={
+                  <p>
+                    <FormattedMessage
+                      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.unableToUseModuleHelpMessage"
+                      defaultMessage="These supplied configurations can be used in {appName}."
```

Comment:
Maybe something like `These jobs can be created in the {appName} app.`?

Reply:
As discussed offline, note that the Logs jobs can't be created in the Serverless oblt project, so we'll need to remove the text about using the jobs in the Logs app on serverless.

Reply:
Is it possible to add in the check for serverless, and remove the text for 'These supplied configurations can be used in Logs' if on serverless?

<img width="804" alt="Screenshot 2024-09-05 at 16 20 51" src="https://github.com/user-attachments/assets/ec392028-c675-42b9-b48f-28a951a83570">

Reply:
@peteharverson - for serverless, the tags have been updated to only show compatible modules so there's no need to do anything on the browserside to check.

For example, for security project those won't show up as they are filtered out by tag in the modules endpoint.
![image](https://github.com/user-attachments/assets/fc1aeecf-0cad-44a2-bda1-9129c028c153)

Reply:
Removed copy in oblt 

![image](https://github.com/user-attachments/assets/1fb05a39-1927-4108-b997-4fa4c66b14dc)